pages/18f/how-18f-works/contractors.md

```diff
@@ -63,6 +63,8 @@ and select the equipment that is needed.
 ## See also
 
 - TTS-specific materials:
+  - [Ethics + Contractors in the Workplace slide deck](https://docs.google.com/presentation/d/1X74B0ZX0yrFxY1ywmC4_EvcZiEQi_97uZ1FUcJPcBUU/edit#slide=id.gf0514cfed4_7_0)
```

Comment:
I lack access to this document; you might want to adjust permissions

Reply:
@jskinne3 Oh, great catch! I used to have access to it, but now I too seem to be missing access ....  I believe Brittney Chappell is the owner of the deck. I'll follow up.

Reply:
@jskinne3 Alrighty, after following up with Brittney on permissions, I'm seeing that the deck is viewable GSA-wide now; are you seeing the same thing?

Reply:
Yes 👍